psi4/driver/procrouting/diis.py

```diff
@@ -1,8 +1,11 @@
 from enum import Enum
+from itertools import product
+
 from psi4 import core
 from psi4.driver import psifiles as psif
 
 import numpy as np
+from scipy.optimize import minimize
```

Comment:
Maybe it's better to add this to the dependencies on CI? At the moment, `scipy` is installed along with `adcc`...

Reply:
That'd probably make tests pass, but at the moment, `scipy` isn't a dependency for core Psi, so we'd still have a dependency that won't be conda-installed in general.